noxfile.py

```diff
@@ -88,3 +88,11 @@ def build(session: nox.Session) -> None:
     """Build an SDist and wheel."""
     session.install("build")
     session.run("python", "-m", "build")
+
+
+@nox.session(reuse_venv=True, python="3.8", default=False)
+def disassemble(session: nox.Session) -> None:
+    """Disassemble run."""
+    session.install("-e", ".[test-extras]")
+    session.posargs.extend(["-ra", "-m", "dis"])
```

Comment:
Can skip "-ra" as it is added here https://github.com/scikit-hep/vector/blob/893ede63861502fd4c4f4111ec8548fa608a1dde/pyproject.toml#L211-L218

```suggestion
    session.posargs.extend(["-m", "dis"])
```